frontend/app/src/comps/ManagedInterestRateField/DelegateModal.tsx

```diff
@@ -0,0 +1,373 @@
+import { useState, ReactNode } from "react";
+import { getRedemptionRisk } from "@/src/liquity-math";
+
+
+import type { Address, CollIndex, Delegate } from "@/src/types";
+
+import { Amount } from "@/src/comps/Amount/Amount";
+import content from "@/src/content";
+import { fmtnum, formatRedemptionRisk } from "@/src/formatting";
+import { useInterestBatchDelegate } from "@/src/subgraph-hooks";
+import { LinkTextButton } from "@/src/comps/LinkTextButton/LinkTextButton";
+import { riskLevelToStatusMode } from "@/src/uikit-utils";
+import { css } from "@/styled-system/css";
+import {
+  AddressField,
+  Button,
+  IconCopy,
+  Modal,
+  StatusDot,
+  TextButton,
+} from "@liquity2/uikit";
+import { MiniChart } from "./MiniChart";
+
+const URL_WHAT_IS_DELEGATION =
+  "https://docs.liquity.org/v2-faq/redemptions-and-delegation#what-is-delegation-of-interest-rates";
```

Comment:
https://docs.nerite.org/docs/user-docs/redemption-and-delegation#what-is-delegation-of-interest-rates